.circleci/config.yml

```diff
@@ -287,25 +287,6 @@ workflows:
             - vulnerability_scan_and_monitor
           context:
             - hmpps-common-vars
-      - hmpps/deploy_env:
```

Comment:
Why is dev going from the ci config?

Reply:
I duplicated it in a previous PR thinking we would do a gradual switchover

This PR removes deploy_dev_newcluster and keeps deploy_dev as one deploy to the new cluster instead